crates/op-steel/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "risc0-op-steel"
+description = "Optimism abstraction for Steel."
+version = "0.1.0-alpha.1"
```

Comment:
What is a good versioning for op-steel?
Should we also mark it as unstable or something.

Reply:
I think marking it as it's own `0.x` version like this makes sense to me. Calling it `0.x` signals that it's unstable.